### PR TITLE
Benji/jib 57 add search

### PR DESCRIPTION
**Pull request by Benjamindodgson**

### CODE REVIEW

File: Shared/UI/View Controllers/BlurredViewController.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//  BlurredViewControllerf.swift
+//  Jibber
+//
+//  Created by Benji Dodgson on 9/20/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class BlurredViewController: FullScreenViewController {
```
Comment (Benjamindodgson) [inline Shared/UI/View Controllers/BlurredViewController.swift:11]: Added this as a base class.

File: iOS/Flows/Archives/ArchiveViewController.swift [hunk 2]
```diff
@@ -24,41 +25,107 @@ class ArchiveViewController: ViewController {
 
     private(set) var channelListController: ChatChannelListController?
 
+    lazy var segmentedControl: UISegmentedControl = {
```
Comment (Benjamindodgson) [inline iOS/Flows/Archives/ArchiveViewController.swift:28]: Search isn't available for messages yet, so just using a segment control

File: iOS/Flows/Archives/ArchiveViewController.swift [hunk 3]
```diff
@@ -74,7 +141,8 @@ class ArchiveViewController: ViewController {
     private func getInitialSnapshot() -> NSDiffableDataSourceSnapshot<ArchiveCollectionViewDataSource.SectionType,
                                                                       ArchiveCollectionViewDataSource.ItemType> {
         var snapshot = self.dataSource.snapshot()
-
```
Comment (Benjamindodgson) [inline iOS/Flows/Archives/ArchiveViewController.swift:77]: Stupid Xcode formatting. Not sure how to fix this.

File: iOS/Flows/Conversation/Input/InputAccessoryView+Animation.swift [hunk 4]
```diff
@@ -1,5 +1,5 @@
 //
-//  MessageInputAccessoryView+Animation.swift
+//  InputAccessoryView+Animation.swift
```
Comment (Benjamindodgson) [inline iOS/Flows/Conversation/Input/InputAccessoryView+Animation.swift:2]: This was named wrong.

**Review by MartinJibber — APPROVED** [review]
Nice looks good.

**Pull request merged**